docs/mesos/ManualInstallCalicoMesos.md

```diff
@@ -0,0 +1,80 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Manually Install Calico and Mesos on Agent
+This guide explains how to add Calico networking to a Mesos slave.
+Specifically, we will add the Calico plugin to perform networking
+for the unified containerizer.
+- For information on the Docker Containerizer, see
+[Docker Containerizer Guide](UsageGuideDockerContainerizer.md).
+
+
+## Prerequisites
+### Mesos-Slave with Mesos and Netmodules installed
+This guide will focus on only the Calico components of a
+calico-mesos-slave. You must have a slave with net-modules
+activated. See the [Manual Net-Modules install Guide](net-modules/README.md).
+
+### Docker
+Calico's core services are run in a Docker container, so we'll need
+Docker installed on every Agent in the cluster.
+[Follow Docker's Centos installation guide](https://docs.docker.com/engine/installation/centos/)
+for information on how to get Docker installed.
+
+### Running etcd instance
+To run Calico, you'll need a running etcd store.
+See the [Cluser Preparation guide](MesosClusterPreparation.md)
+for information on how to quickly get one running.
+
+## Install Calico-Mesos Components
+The following calls will download the necessary Calico components.
+Ensure you've created the `/calico` directory first (`mkdir -p /calico`).
+
+  1. Get the `calico_mesos` plugin binary
+
+  ```
+  curl -L -o /calico/calico_mesos https://github.com/projectcalico/calico-mesos/releases/download/v0.1.5/calico_mesos
+  chmod +x /calico/calico_mesos
+  ```
+
+  2. Download the `modules.json` file
```

Comment:
Perhaps install in the modules.json file..... you don't really enable the file, and these instructions aren't telling you to load it.